shared/errorcodes.js

```diff
@@ -0,0 +1,40 @@
+const errorCodes = {
+	ACCOUNT_NOT_FOUND: {
+		enum: "ACCOUNT_NOT_FOUND",
```

Comment:
I don't get the point of this `enum` property, sounds like code duplication to me.
Is it used client-side?

Reply:
As bob said, we could just send a localization string ike '#ACCOUNT_NOT_FOUND' to the client and let that deal with it

Reply:
Yes, the plan is to prepend a '#' to this string and then localize it